netmap/attributes.go

```diff
@@ -118,6 +118,8 @@ var errNoSubnets = errors.New("no subnets")
 // IterateSubnets iterates over all subnets the node belongs to and passes the IDs to f.
 // Handler must not be nil.
 //
+// Subnet attributes are expected to be key-unique, otherwise undefined behavior.
```

Comment:
What about returning an error on duplication and/or process first only?

Reply:
Process first only doesn't fit definitely. It is expected that control and observance of uniqueness will be done in advance.